tests/rptest/services/redpanda.py

```diff
@@ -726,6 +726,9 @@ def get_node_cpu_count(self):
     def get_node_disk_free(self):
         pass
 
+    def lsof_node(self, node: ClusterNode, filter: Optional[str] = None):
+        pass
+
```

Comment:
nit: can be squashed with prev commit; actually all "add method to ovverride" commits can be squashed together

Reply:
sorry, i didn't catch this comment because i went straight for the merge button. thanks for the review! :twisted_rightwards_arrows: